firebase.json

```diff
@@ -23,7 +23,7 @@
       { "source": "/\\)", "destination": "/", "type": 301 },
       { "source": "/+", "destination": "https://plus.sandbox.google.com/+dartlang", "type": 301 },
       { "source": "/+dart2js", "destination": "/tools/dart2js", "type": 301 },
-      { "source": "/+isolates", "destination": "https://dart-lang.github.io/observatory/isolate.html", "type": 301 },
+      { "source": "/+isolates", "destination": "https://medium.com/dartlang/dart-asynchronous-programming-isolates-and-event-loops-bffc3e296a6a", "type": 301 },
```

Comment:
I'm curious where this link is referred from. I'm guessing @kwalrath isn't updating the medium article?

Reply:
Yeah I'm not sure of any references to the redirect, but the article seemed the best destination for now. I believe there may be new isolate related content in the future though which could perhaps replace it.